test/relations.test.js

```diff
@@ -2776,10 +2953,14 @@ describe('relations', function() {
     it('should find record on scope with promises', function(done) {
       Passport.findOne()
       .then(function(p) {
-        p.personId.should.eql(personCreated.id);
+        if (!isCassandraConnector) {
```

Comment:
Yes, Person.findById(personCreated.id) instead of Person.findOne would return the specific one.
However,  Passport.findById(personCreated.id) does not work because personCreated.id is not a primary key for Passport model.